src/main/java/cn/nukkit/entity/ai/MobAIUnion.java

```diff
@@ -0,0 +1,55 @@
+package cn.nukkit.entity.ai;
+
+import cn.nukkit.entity.Entity;
+import cn.nukkit.math.Vector3;
+
+/**
+ * You can combine different AI's together
+ *  - The fist MobAI to return a valid target is used
+ */
+public class MobAIUnion implements MobAI {
```

Comment:
Well I assume that you can create some class like `MobAISet` for the AI for a mob.